### PR TITLE
verify non empty raw tags

### PR DESCRIPTION
**Pull request by ibrahni**

### CODE REVIEW

File: src/test/java/com/baeldung/selenium/common/ArticlesUITest.java [hunk 1]
```diff
@@ -496,7 +496,28 @@ public final void givenAllArticles_whenAnArticleLoads_thenItDoesNotLinkToOldJava
             triggerTestFailure(badURLs);
         }
     }
-    
+
+    @Test
+    public final void givenAllArticles_whenArticleLoads_thenArticleHasNoEmptyRawBlock(){
+        log(GlobalConstants.givenAllArticles_whenArticleLoads_thenArticleHasNoEmptyRawBlock);
+        recordExecution(GlobalConstants.givenAllArticles_whenArticleLoads_thenArticleHasNoEmptyRawBlock);
```
Comment (asif-anwar) [inline src/test/java/com/baeldung/selenium/common/ArticlesUITest.java:503]: Let's move the `recordExecution()` inside the do loop. This is actually a very recent change we did in JAVA-11289.

File: src/main/java/com/baeldung/site/SitePage.java [hunk 2]
```diff
@@ -61,6 +61,10 @@ public List<WebElement> findEmptyCodeBlocks(){
         return this.getWebDriver().findElements(By.xpath("//code[((.='\\u00a0')  or (normalize-space(.)=''))]"));
     }
 
+    public List<WebElement> findEmptyRawBlocks(){
+        return this.getWebDriver().findElements(By.xpath("//raw[((.='\\u00a0')  or (normalize-space(.)=''))]"));
```
Comment (asif-anwar) [inline src/main/java/com/baeldung/site/SitePage.java:65]: I think it's looking for empty [raw] tags. Actually, it's not about empty tags. We shouldn't have the [raw] tag at all in the page. 
The article should be flagged if it has either [raw] or [/raw] string in the page source. 
I think we check this by getting the pageSource using page source WebDriver.getPageSource()

File: src/test/java/com/baeldung/selenium/common/ArticlesUITest.java (hunk 1 above)
Comment (ibrahni) [reply src/test/java/com/baeldung/selenium/common/ArticlesUITest.java]: done

File: src/main/java/com/baeldung/site/SitePage.java (hunk 2 above)
Comment (ibrahni) [reply src/main/java/com/baeldung/site/SitePage.java]: @asif-anwar , changes done  , now I am checking if the raw tag is used in the source or not

**Pull request merged**